src/rt-app.c

```diff
@@ -942,6 +950,10 @@ static void set_thread_priority(thread_data_t *data, sched_data_t *sched_data)
 					" period %lu", data->ind,
 					policy_to_string(sched_data->policy), sched_data->period,
 					sched_data->runtime, sched_data->deadline);
+			log_ftrace(ft_data.marker_fd, FTRACE_ATTRS,
+				   "rtapp_attrs: event=policy policy=dl runtime=%lu deadline=%lu period=%lu",
+				   sched_data->prio, sched_data->runtime,
+				   sched_data->deadline, sched_data->period);
```

Comment:
This has a different signature than above ie: you print more than just policy and prio. Will it create confusion since IIUC the event name will always be rtapp_attrs when viewed in pandas but we can end up with 2 different set of columns depending on the policy we are setting?

Reply:
Your right, we will end up with multiple columns but that's intentional.

What I'm trying to do is to cover cases where we have only one clamp updated.
Once we merge the rows in Pandas we will end up with `NaN` values for the clamp not updated. That will allow to easily do a forward propagation of any possible previous update to overwrite the `NaN`s.

Otherwise, to always report both values we will have to either track then in rt-app or always read them before writing. Preferences?

Reply:
No preference. As long as you've thought about it and have a clear idea on how to deal with it I'm happy with it. We can evolve later if the need ever arises.